spec/models/user_spec.rb

```diff
@@ -1,3 +1,4 @@
+require './features/support/omni_auth_fixtures'
```

Comment:
This fixture is probably already included in rails helper since it resides in the... Oh, you have it in the `features` folder... Hmm.. Any way, do not include it here, but rather in `rails_helper`